src/__tests__/entity.update.unit.test.ts

```diff
@@ -409,10 +409,9 @@ describe('update', () => {
       ExpressionAttributeNames,
       ExpressionAttributeValues
     } = TestEntity.updateParams({
-      pk: 'test-pk',
-      sk: 'test-sk',
+      email: 'test-pk',
```

Comment:
It should have thrown an ts-error earlier, shouldn't it ? 🤔

Reply:
Yeah, I thought so it, very odd.